### DISCUSSION TITLE
Want to use TailwindCSS in place of `.scss` styles

### DISCUSSION THREAD

**Discussion by m-naeem66622** [payloadcms/payload#5028] [Feature Requests & Ideas]
## My words about payload
Payload is just mind blowing thing for me to create e-commerce website in short time without giving headche to developers (in case it's me) changing the stuff or updating the stuff again and again... But one thing is that the template for e-commerce is using sass to style webapges. Trust me i worked with both sass and tailwindcss. But i am good at styling webpages with tailwindcss because i had used almost three component libraries with that including `nextui` , `shadcn` and `daisyui`. Beacause it is easy peasy task to add a new theme or cutomizing th existing theme.. The way for cusotmizing with sass is more difficult for me as we have to adjust different color shades manually (in `src/app/_css/colors.scss`

## Lets get to the point
I want to use some tailwindcss component library to style or create components. So anyone can give option to setup with Tailwindcss styles or something else (by default which is `.scss` right now.

**Comment by m-naeem66622** [comment]
## Continuing the previous
So just like nextjs give option to setup ***"with TypeScript or not"*** and ***with Tailwindcss or not***. Similarly with Payload I am wondering there should be an option to setup template with ***Tailwindcss or not***. Might be related to ***Yes or No***.

**Comment by Ezejaemmanuel** [comment]
me too...i prefer tailwind to sass...i hate sass

**Comment by TimBar42** [comment]
Same, hate sass, wish I could just work with Tailwind CSS. Hope Payload implements that in the coming updates

**Comment by lisvan06** [comment]
Until the option comes to us we can use this implementation: [https://www.youtube.com/watch?v=RhoWLkIrMN0].

Note: Add the route of the tailwind.css file to the in buildConfig in payload.config.ts
```
custom: {
    css: path.resolve(__dirname, '../tailwind.css'),
  }
```

Works for me!!

**Comment by mflisikowski** [comment]
https://github.com/payloadcms/payload/issues/8878
